Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeDrawingAreaProxyMac.mm

```diff
@@ -241,34 +247,61 @@ explicit RemoteLayerTreeDisplayLinkClient(WebPageProxyIdentifier pageID)
     return animation;
 }
 
+static RetainPtr<CABasicAnimation> fillFowardsAnimationWithKeyPathAndValue(NSString *keyPath, NSValue *value)
+{
+    RetainPtr animation = fillFowardsAnimationWithKeyPath(keyPath);
+    [animation setFromValue:value];
+    [animation setToValue:value];
+    return animation.autorelease();
```

Comment:
`.autorelease()` to return a `RetainPtr`?

Reply:
Whoops! Yeah, that should just be `return animation;`